src/include/sof/debug/debug.h

```diff
@@ -126,7 +127,8 @@
 (									\
 	(IS_ENABLED(CONFIG_DEBUG_LOCKS) ? SOF_IPC_INFO_LOCKS : 0) |	\
 	(IS_ENABLED(CONFIG_DEBUG_LOCKS_VERBOSE) ? SOF_IPC_INFO_LOCKSV : 0) | \
-	(IS_ENABLED(CONFIG_GDB_DEBUG) ? SOF_IPC_INFO_GDB : 0)		\
+	(IS_ENABLED(CONFIG_GDB_DEBUG) ? SOF_IPC_INFO_GDB : 0) |		\
+	(IS_ENABLED(CONFIG_CAVS) ? SOF_IPC_INFO_IMR_RESTORE : 0)		\
```

Comment:
If the host decide what happens, is there a way for the host to tell the DSP to skip the status save in IMR?